tools/BitFunnel/src/REPL.cpp

```diff
@@ -183,7 +183,7 @@ namespace BitFunnel
             << std::endl;
 
         environment.StartIndex();
-        environment.SetShards(0, environment.GetIngestor().GetShardCount());
+        environment.SetShards(0, environment.GetIngestor().GetShardCount() - 1);
 
```

Comment:
It's interesting that we never encountered this bug before.

Reply:
That's because I just introduced in the prior commit.